Dockerfile

```diff
@@ -20,14 +20,44 @@ RUN ["bash", "-c", "curl -fsSL https://deb.nodesource.com/setup_16.x | bash -"]
 RUN apt install nodejs -y
 RUN npm install -g yarn
 
-# Install App
+# Dashboard
+WORKDIR /dashboard
+
+COPY dashboard/package.json package.json
+COPY dashboard/yarn.lock yarn.lock
+RUN yarn install
+
+COPY dashboard/assets assets
+COPY dashboard/public public
+COPY dashboard/tsconfig.json tsconfig.json
+COPY dashboard/tailwind.config.js tailwind.config.js
+COPY dashboard/craco.config.js craco.config.js
+COPY dashboard/src src
+ENV REACT_APP_IASQL_ENV=local
```

Comment:
Perhaps we should move the `IASQL_ENV` definition below up here and then have this use that as the default value if not specified? (Not a blocker on this PR, thinking out lout)